optimum/habana/transformers/models/wav2vec2/modeling_wav2vec2.py

```diff
@@ -300,58 +369,84 @@ def gaudi_wav2vec2_encoder_forward(
     )
 
 
-def gaudi_wav2vec2_forward(
+_HIDDEN_STATES_START_POSITION = 2
```

Comment:
Maybe we can import `_HIDDEN_STATES_START_POSITION` from `transformers.models.wav2vec2.modeling_wav2vec2`?

Reply:
Ok, let me try doing that.